crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1243,11 +1243,10 @@ pub async fn generate_genesis_system_object<
     let genesis_digest = genesis_ctx.digest();
     let mut temporary_store =
         AuthorityTemporaryStore::new(store.clone(), InputObjects::new(vec![]), genesis_digest);
-    let pubkeys: Vec<Vec<u8>> = committee
-        .expanded_keys
-        .values()
-        .map(|pk| pk.to_bytes().to_vec())
-        .collect();
+    let mut pubkeys = Vec::new();
+    for name in committee.voting_rights.keys() {
+        pubkeys.push(committee.public_key(name)?.to_bytes().to_vec());
```

Comment:
When (and why?) would the name lookup fail in practice?

Reply:
I probably won't fail in practice. The only way for it to fail is when the committee information is loaded from the store which happens to contain invalid public keys

Reply:
Oh I see, so if we don't panic what should we do? We cannot make progress anyway, right?